app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminClickSaveOnProductsMassAttributeUpdateActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClickSaveOnProductsMassAttributesUpdateActionGroup">
+        <annotations>
+            <description>Clicks on 'Save' button on products mass attributes update page.</description>
+        </annotations>
+
+        <click selector="{{AdminEditProductAttributesSection.Save}}" stepKey="save"/>
```

Comment:
If you save something, is much better to make sure the page is loaded afterwards `waitForPageLoad`, including a success/error message if that's the case.
Also please remove the empty rows in your changed files.

Reply:
@eduard13 thank you for the advice - this has been addressed.